src/shred_collector/shred_processor.zig

```diff
@@ -62,13 +62,15 @@ pub fn runShredProcessor(
         }
         metrics.insertion_batch_size.observe(shreds.items.len);
         metrics.passed_to_inserter_count.add(shreds.items.len);
-        _ = try shred_inserter.insertShreds(
-            shreds.items,
-            is_repaired.items,
-            leader_schedule,
-            false,
-            null,
-        );
+        _ = shred_inserter_;
+        _ = leader_schedule;
+        // _ = try shred_inserter.insertShreds(
+        //     shreds.items,
+        //     is_repaired.items,
+        //     leader_schedule,
+        //     false,
+        //     null,
+        // );
```

Comment:
Can we uncomment this?

Reply:
I will uncomment this after running final tests once other changes are addressed

Reply:
cf58e76